routes/payout.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const router = express.Router();
+const User = require("../models/user.model");
+
+const { baseURL } = require('../config/index');
+
+const stripeSecretKey = process.env.STRIPE_SECRET_KEY
+const stripe = require('stripe')(stripeSecretKey);
+
+// Route to put user into Stripe connection flow
+router.post('/setup', async(req, res) => {
+
+  const { email } = req.body;
+
+  try{
+      const account = await stripe.accounts.create({
+          type: 'express',
+          country: 'US',
+        });
+
+      const accountLink = await stripe.accountLinks.create({
+      account: account.id,
+      refresh_url: `${baseURL}/myAccount`,
+      return_url: `${baseURL}/myAccount`,
+      type: 'account_onboarding',
+      });
+
+      const redirectURL = accountLink['url'];
+
+      const user = await User.findOne({ email });
```

Comment:
What happens if the email provided is not associated with a valid user? We need a validation check here that returns a 400 on invalid email to ensure that the next line (line 31) does not break.

Reply:
Another option is to just use `findOneAndUpdate` to bypass the potential issues on line 31. If we choose to use `findOneAndUpdate`, we should still add a validation check to see if the email is valid or not though.